src/shapes/SurfaceEllipse.js

```diff
@@ -212,6 +212,19 @@ define([
             }
         };
 
+        // Internal use only. Intentionally not documented.
+        SurfaceEllipse.prototype.getReferencePosition = function () {
+            return this.center;
+        };
+
+        // Internal use only. Intentionally not documented.
+        SurfaceEllipse.prototype.moveTo = function (globe, position) {
+            var locations = [];
+            locations.push(new Location(this.center.latitude, this.center.longitude));
```

Comment:
Could the center property be used directly instead of creating a new location object?